tools/travis_script_osx

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash -ex
+#
+# this script is run on travis for the script stage of mac builds
+#
+
+QMAKE=${QMAKE:-qmake}
+LUPDATE=${LUPDATE:-lupdate}
+LRELEASE=${LRELEASE:-lrelease}
+MACDEPLOYQT=${MACDEPLOYQT:-macdeployqt}
+VERSIONID=${VERSIONID:-$(git rev-parse --short HEAD)}
+
+# build and test the CLI
+$QMAKE GPSBabel.pro && make && make check
+
+# build the GUI
+pushd gui
+$QMAKE app.pro && make
+$LUPDATE app.pro
+$LRELEASE app.pro
+popd
+
+# package the app
+mkdir -p gui/objects/GPSBabelFE.app/Contents/MacOS/translations
+cp gui/*.qm  gui/objects/gpsbabelFE.app/Contents/MacOS/translations
+cp GPSBabel gui/objects/GPSBabelFE.app/Contents/MacOS/gpsbabel
+# this help seems usless, why bother?
```

Comment:
Agreed. Start of a bad idea that was never taken further. Please drop it.

Reply:
I deleted the help/gpsbabel.html from the mac dmg, the windows installer, and makelinuxdist.